operations/research-and-development/product/growth/ab-testing.md

```diff
@@ -0,0 +1,119 @@
+  * This will also be a contributing factor in determining the required sample size.
+  * Smaller relative change targets require larger sample sizes in order to achieve the same confidence and statistical power as larger relative change targets with smaller sample sizes.
+    * Think about it this way: The smaller the relative change you are trying to detect → The narrower the confidence interval must be to detect this change → The primary method to narrow confidence intervals is to increase the sample size.
+    * Many A/B tests will not attain the minimum required relative change or will never reach the required sample size in order to detect it, which is why it’s important to identify targets and forecast timelines in advance. 
+
+The success criteria of an A/B test is based on one to two clearly defined measures, and how an intervention hopes to influence them i.e. increase, decrease, or change (regardless of direction). Clearly defined measures ensure the ability to quantify the impact of interventions, thereby allowing you to measure the success or failure of a test.
+
+Examples of valid success criteria metrics/measures:
+1. First 14-Day Invite Member Conversion Rate: Proportion of total users to invite new members w/in the first 14 days of first activity
+    * I.e. the percentage of total users that trigger an invite member event w/in 14 days of the user’s first active timestamp.
+2. Cloud Workspace Creation Conversion Rate: Proportion of unique visitors to the Cloud Trial Signup Page that successfully create a cloud workspace
+
+### Formulating Hypotheses
+Once you’ve defined the target measure and established the success criteria, you must determine the [null and alternative hypotheses](https://www.youtube.com/watch?v=_Qlxt0HmuOo) of the test. The null hypothesis is always that the intervention will have no effect on the target measure i.e. Test Outcome = Control Outcome. The alternative hypothesis determines the type of change we’re testing for. If the alternative hypothesis is that the intervention will affect the measure, regardless of direction (Test Outcome ≠ Control Outcome), then the test is 2-sided. If the alternative hypothesis is that the intervention will increase (or decrease) our target measure (Test Outcome > | < Control Outcome) then this is a 1-sided test.
+
+Example of valid Null & Alternative Hypothesis:
+* H𝞱 (Null Hypothesis): Adding an invite members icon to the banner will have no effect on the First 14-Day Invite Member Conversion Rate. 
+  * 𝞱 = 𝞱0
+* Ha (One-Sided Alternative Hypothesis): Adding an invite members icon to the banner will increase the First 14-Day Invite Member Conversion Rate. 
+  * 𝞱 > 𝞱0
+* Ha (Two-Sided Alternative Hypothesis): Adding an invite members icon to the banner will change (either direction) the First 14-Day Invite Member Conversion Rate. 
+  * 𝞱 ≠ 𝞱0
+
+
+### Minimum Detectable Effect Size (MDE)
```

Comment:
Is this the same as point 4 in the success criteria section?

Reply:
Yeah, we can consolidate the 2 sections into a single one below the Success Criteria as a subheader since it's a portion of defining the success criteria.

Reply:
Removed and pointed to section below. @jasonblais